### PR TITLE
Skirollytelling

### PR DESCRIPTION
**Pull request by trangdata**

### CODE REVIEW

File: website/skirollytelling/_extensions/qmd-lab/closeread/closeread.lua
Comment (dhimmel) [inline website/skirollytelling/_extensions/qmd-lab/closeread/closeread.lua:1]: is this file copied from somewhere? If so can we include the location in the header comment? Or possibly even load via URL?

Comment (dhimmel) [reply website/skirollytelling/_extensions/qmd-lab/closeread/closeread.lua:1]: Ah I see this whole directory is added by `quarto add qmd-lab/closeread` as per https://closeread.dev/

Comment (dhimmel) [reply website/skirollytelling/_extensions/qmd-lab/closeread/closeread.lua:1]: we can install it from commit like:

```
quarto add https://github.com/qmd-lab/closeread/archive/e3645070dd668004056ae508d2d25d05baca5ad1.zip
```

but it's going one directory higher

**Comment by trangdata** [comment]
I will use these images in index.md.

dartmouth
openskimap

File: .github/workflows/analyze.yaml [hunk 1]
```diff
@@ -33,6 +33,13 @@ jobs:
       uses: r-lib/actions/setup-renv@v2
       with:
         working-directory: website/skirolly/
+    - name: Make skirolly plots
+      working-directory: website/skirolly/
+      shell: Rscript {0}
+      run: |
+        source("01.data.R")
+        source("02.setup.R")
+        source("03.plot.R")
```
Comment (dhimmel) [inline .github/workflows/analyze.yaml:42]: perhaps like `Rscript 01.data.R`

File: website/skirolly/03.plot.R [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# source("02.setup.R")
```
Comment (dhimmel) [inline website/skirolly/03.plot.R:1]: no imports?

**Pull request merged**